cmd/helm/upgrade.go

```diff
@@ -266,6 +267,10 @@ func (u *upgradeCmd) run() error {
 		return prettyError(err)
 	}
 
+	if ch.Metadata.Deprecated {
+		fmt.Fprintln(os.Stderr, "WARNING: This chart is deprecated")
```

Comment:
Same comment as other location

Reply:
@mattfarina , are you good with @bacongobbler 's comment?

@bacongobbler , thanks for the explanation.

Reply:
Resolve, per @mattfarina 's comment above.